cromwell_tools/cromwell_tools.py

```diff
@@ -214,3 +226,86 @@ def read_local_file(path):
     with open(path) as f:
         contents = f.read()
     return contents
+
+
+def _content_checker(regex, content):
+    """
+    Helper function to check if a string is obeying the rule described by a regex string or not.
+
+    :param str regex: A regex string defines valid content.
+    :param str content: A string to be validated.
+
+    :return str: A string of error message if validation fails, or an empty string if validation succeeds.
+    """
+    try:
```

Comment:
Would be helpful to have a comment here explaining that (if I understand right) we expect the thing in the try block to succeed in Python 3 and fall back on the thing in the except block for Python 2.

Reply:
Yes, that is better for readability. Will do. Thanks!